runtime/tests/checker/arrays_dictionaries_test.go

```diff
@@ -753,6 +753,72 @@ func TestCheckInvalidArrayRemoveLastFromConstantSized(t *testing.T) {
 	assert.IsType(t, &sema.NotDeclaredMemberError{}, errs[0])
 }
 
+func TestCheckArrayIndexOf(t *testing.T) {
+
+	t.Parallel()
+
+	_, err := ParseAndCheck(t, `
+      fun test(): Int? {
+          let x = [1, 2, 3]
+					return x.firstIndex(of:2)
+      }
+    `)
+
+	require.NoError(t, err)
+}
+
+func TestCheckArrayFirstIndexDoesNotExist(t *testing.T) {
+
+	t.Parallel()
+
+	_, err := ParseAndCheck(t, `
+      fun test(): Int? {
+          let x = [1, 2, 3]
+					return x.firstIndex(of:5)
+      }
+    `)
+
+	require.NoError(t, err)
+}
+
```

Comment:
This test is a duplicate of the one above, the argument passed to the function does not matter for the static analysis.

```suggestion
```